src/docs/patterns/navigation.mdx

```diff
@@ -1,70 +1,311 @@
+<ImageGrid images={[Banner]} regularWidth />
 
-<MaterialDesignDescription
-    title={'Understanding Navigation'}
-    url={'https://material.io/design/navigation/understanding-navigation.html#types-of-navigation'}
-    description={`Read about Material Design's navigation fundamentals.`}
+Navigation refers to how users access the different areas of your application and move from screen to screen. The most common forms of primary navigation are drawers and tabs.
+
+<React.Fragment>
+    <MaterialDesignDescription
+        title={'Understanding Navigation'}
+        url={'https://material.io/design/navigation/understanding-navigation.html#types-of-navigation'}
+        description={`Read about Material Design's navigation fundamentals.`}
+    />
+    <Divider />
+</React.Fragment>
+
+# Drawer Navigation
+
+A navigation drawer is a panel that is attached to the side of your application. These typically contain a [List](/patterns/lists) of primary pages in your application for the user to access quickly.
+
+<ImageGrid
+    images={[NavigationBreakdownImg]}
+    caption={['1. Header', '2. Subheader', '3. Body', '4. Footer']}
+    regularWidth
+/>
+
+Most drawers have four main sections:
+
+-   **Header**: contains the name of the company, name of the product, or a logo
+-   **Subheader**: an optional section that can be used to support custom content, such as filtering or search
+-   **Body**: contains a list of primary pages that users can navigate to
+-   **Footer**: contains the company logo, version number, and trademark information
+
+There are several different styles of drawer navigation, each with slightly different behavior.
+
+<React.Fragment>
+    <DemoCard
+        repository={'Drawer'}
+        angular={'https://pxblue-components.github.io/angular/?path=/story/components-drawer--within-a-drawer-layout'}
+        react={'https://pxblue-components.github.io/react/?path=/story/components-drawer--in-drawer-layout'}
+    />
+    <MaterialDesignDescription
+        title={'Navigation Drawer '}
+        url={'https://material.io/components/navigation-drawer/'}
+        style={{ marginRight: 0 }}
+    />
+</React.Fragment>
+
+## Permanent
+
+A permanent drawer is always visible and always open. The permanent drawer is typically full-width, as opposed to the rail variant (see below). Use this variation if you have:
+
+-   five or more top-level pages
+-   a nested navigation hierarchy
+-   enough space to justify it (typically desktop-only applications)
+
+<ImageGrid images={[NavigationPermanentImg]} caption={['A permanent drawer on desktop.']} regularWidth />
+
+## Rail
+
+A navigation rail is a permanently collapsed drawer. The rail can show less information than a standard drawer but has the advantage of a more compact, space-saving package. Use this variation if you have:
+
```

Comment:
Should we say anything about the difference between condensed and normal rail?

Reply:
Mostly a matter of preference. Or we can push people to use the regular labeled version unless they are confident that their users can infer the icons’ meaning with no problem.

Reply:
I'm okay with keeping it simple. I just noticed that we didn't say anything about it except in the caption of the image. Might be nice to add a sentence somewhere, e.g.: "The navigation rail can be regular (with text labels) or condensed (smaller area without labels)."

Reply:
I didn't see this before I hit that merge button 🙈 

Time to abuse your admin authority.